firmware/arduino/chessboard/circleFunction.ino

```diff
@@ -0,0 +1,224 @@
+int pulsesPerSlope[NUM_CIRCLES][NUM_SLOPES_PER_QUARTER_CIRCLE];
+
+enum SlopeToIndex
+{
+    SLOPE_HORIZONTAL = 0,
+    SLOPE_EIGHTH = 1,
+    SLOPE_QUARTER = 2,
+    SLOPE_HALF = 3,
+    SLOPE_ONE = 4,
+    SLOPE_TWO = 5,
+    SLOPE_FOUR = 6,
+    SLOPE_EIGHT = 7,
+    SLOPE_VERTICAL = 8
+};
+
+
+// Makes a full circle of the given size (0=largest, NUM_CIRCLES-1=smallest) 
+// starting from the given quarter (0=top, 1=left, 2=bottom, 3=right).
+// calculatePulsesPerSlope() must be called before makeCircle()
+void makeCircle(int circle, int firstQuarter)
+{
+    bool firstPass = true;
+    int slopeIndex;
+
+    setScale(xMotor, EIGHTH_STEPS);
+    digitalWrite(xMotor[STEP_PIN], LOW);
+
+    for (int quarterCircle = firstQuarter; quarterCircle != firstQuarter || firstPass; quarterCircle = (quarterCircle + 1) % 4)
```

Comment:
I think changing quarterCircle to quarter is just as clear and makes the line shorter